### PR TITLE
116 kruskal's algo

### PR DESCRIPTION
**Pull request by Piyush-onGIT**
Fixed issue no #116 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Added code for Kruskal's Algorithm in C++, Kindly review the PR @Kumar-laxmi 

## 📷 Screenshots
Screenshot (8)

### CODE REVIEW

File: C++/Graphs/Kruskal_Algo.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    return a.wt < b.wt;
+}
+
+int findPar(int node, vector<int> &parent) {
+    if (node == parent[node]) return node;
+    return parent[node] = findPar(parent[node], parent);
+}
+
+void unionn(int u, int v, vector<int> &parent, vector<int> &Rank) {
+    u = findPar(u, parent);
```
Comment (siddhi-244) [inline C++/Graphs/Kruskal_Algo.cpp:27]: Try to add a comment here for eg why we are finding out parents? How it would help ?

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
Add some one line comments so that the approach is understood well .

**Review by zugzwang03 — APPROVED** [review]

**Review by Rishikavishnoi — APPROVED** [review]

**Review by Sanjanabharadwaj25 — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Mention the issue number of ```Kosaraju_Algorithm.cpp``` and update the title accordingly

**Comment by Kumar-laxmi** [comment]
@Piyush-onGIT Kindly do the required changes within a week or else this PR will be closed

**Pull request closed without merging**